hdkeychain/doc.go

```diff
@@ -26,8 +26,8 @@ Transaction Signing Keys and Payment Addresses
 
 In order to create and sign transactions, or provide others with addresses to
 send funds to, the underlying key and address material must be accessible.  This
-package provides the ECPubKey, ECPrivKey, and Address functions for this
-purpose.
+package provides the ECPubKey and ECPrivKey functions for this purpose.  The
+caller may then create the desired address types as desired.
```

Comment:
You can probably remove the last "as desired" from:

"... desired address types as desired"

Reply:
Agreed.  Updated.